sources/iTermPreferencesBaseViewController.m

```diff
@@ -157,6 +170,10 @@ - (IBAction)settingChanged:(id)sender {
                 [self setInt:[sender selectedTag] forKey:info.key];
                 break;
 
+            case kPreferenceInfoTypeUPopup:
+                [self setUInt:[sender selectedTag] forKey:info.key];
```

Comment:
`-[NSPopupButton selectedTag]` is an `NSInteger`. I think it would be a bug for it to have a negative value, so we should assert that it is positive here.